app/controllers/your-psp/post-flex.controller.test.js

```diff
@@ -0,0 +1,93 @@
+'use strict'
+
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+
+const gatewayAccountFixtures = require('../../../test/fixtures/gateway-account.fixtures')
+
+describe('Post 3DS Flex controller', () => {
```

Comment:
Can a test be added for the connector returning an error for `await connector.updateIntegrationVersion3ds(accountId, INTEGRATION_VERSION_3DS)   `

Reply:
Good idea - done